src/app/core/components/event-card/event-card.component.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { DebugElement } from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+import { MatCardModule } from '@angular/material/card';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Event } from '../../../../backend-client/model/event';
+
+import { EventCardComponent } from './event-card.component';
+
+describe('EventCardComponent', () => {
+  let component: EventCardComponent;
+  let fixture: ComponentFixture<EventCardComponent>;
+
+  const MOCK_EVENT_OBJECT: Event = {
+    name: 'Mock Test Event 1',
+    description: 'This is a event abot mocking',
+    organizationID: 'Chula.Org',
+    tags: ['Hello', 'World'],
+    startDate: ('2020-06-13T13:54:29.808Z' as unknown) as Date,
```

Comment:
There is something wrong about this. But it is not our fault. It should be something wrong in the backend swagger config. Let me talk with backend guys about this.

Reply:
string should not be converted to unknown then to Date